deploy/templates/bicep/ContainerAppSetup.bicep

```diff
@@ -0,0 +1,392 @@
+        {
+          server: '${containerRegistry.name}.azurecr.io'
+          identity: userAssignedMI.id
+        }
+      ]      
+    }
+    template: {
+      containers: [
+        {
+          image: '${containerRegistry.name}.azurecr.io/${normalizationImage}:${imageTag}'
+          name: 'normalization-${timestamp}'
+          env: [
+            {
+              name: 'APPINSIGHTS_INSTRUMENTATIONKEY'
+              value: appInsights.properties.InstrumentationKey
+            }
+            {
+              name: 'EventBatching__FlushTimespan'
+              value: '30'
+            }
+            {
+              name: 'EventBatching__MaxEvents'
+              value: '20'
+            }
+            {
+              name: 'Checkpoint__BlobPrefix'
+              value: 'Normalization'
+            }
+            {
+              name: 'CheckpointStorage__AuthenticationType'
+              value: 'ManagedIdentity'
+            }
+            {
+              name: 'CheckpointStorage__BlobStorageContainerUri'
+              value: '${storageAccount.properties.primaryEndpoints.blob}checkpoint'
+            }
+            {
+              name: 'CheckpointStorage__BlobContainerName'
+              value: 'checkpoint'
+            } 
+            {
+              name: 'TemplateStorage__AuthenticationType'
+              value: 'ManagedIdentity'
+            } 
+            {
+              name: 'TemplateStorage__BlobStorageContainerUri'
+              value: '${storageAccount.properties.primaryEndpoints.blob}template'
+            }
+            {
+              name: 'TemplateStorage__BlobContainerName'
+              value: 'template'
+            }
+            {
+              name: 'InputEventHub__AuthenticationType'
+              value: 'ManagedIdentity'
+            }
+            {
+              name: 'InputEventHub__EventHubNamespaceFQDN'
+              value: eventhubNamespace.properties.serviceBusEndpoint
+            }
+            {
+              name: 'InputEventHub__EventHubConsumerGroup'
+              value: '$Default'
+            }
+            {
+              name: 'InputEventHub__EventHubName'
+              value: 'devicedata'
+            }
+            {
+              name: 'NormalizationEventHub__AuthenticationType'
+              value: 'ManagedIdentity'
+            }
+            {
+              name: 'NormalizationEventHub__EventHubNamespaceFQDN'
+              value: eventhubNamespace.properties.serviceBusEndpoint
+            }
+            {
+              name: 'NormalizationEventHub__EventHubConsumerGroup'
+              value: '$Default'
+            }
+            {
+              name: 'NormalizationEventHub__EventHubName'
+              value: 'normalizeddata'
+            } 
+            {
+              name: 'Template__DeviceContent'
+              value: 'devicecontent.json'
+            }
+          ]
+        }
+      ]
+      scale: {
+        minReplicas: 1
+        maxReplicas: 10
```

Comment:
can we make it just 2 if that meets the basic requirements?

Reply:
Are you referring to the number of minReplicas or are you suggesting there should always only be two replicas?

Reply:
oh, sorry about the confusion. I was thinking max as 2. The reason is we don't want to surprise customer. Or you can add a reminder about the max replica.